rules.txt

```diff
@@ -271,6 +272,7 @@
 /documents/ohscontractattachments/        301  /departments/office-of-homeless-services/documents/standards-policies-and-forms/
 /documents/grief-support-groups-for-drug-overdose-deaths/ 301  /programs/substance-use-prevention-and-harm-reduction/bereavement-support-services/
 /documents/visit-the-medical-examiners-office/ 301  /departments/medical-examiners-office/
+/documents/food-matters-project-documents/ 301  /programs/zero-waste-initiatives/food-waste-reduction/
```

Comment:
Looking good! I'm wondering if we need to worry about the trailing slash on the vanity url. The redirects seem to be working with or without them, but I'm guessing that's a browser change. We should do some testing to see if they are still something we need to be concerned with.